automation/project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ import sbt.Keys._
 object Dependencies {
   val jacksonV = "2.9.0"
 
-  val serviceTestV = "0.11-e923c03"
+  val serviceTestV = "0.12-560c0ce-SNAP"
```

Comment:
I will update to a release version once broadinstitute/workbench-libs#148 merges.